environments/template/group_vars/template.yml

```diff
@@ -623,8 +623,13 @@ haproxy_applications:
 
 haproxy_backend_tls: False
 
-haproxy_sni_ip: "*"
-haproxy_sni_ipv6: "::1"
+haproxy_sni_ip:
+  ipv4: "*"
+  ipv6:  "::1"
+  certs: 
+     - name: star
+       key_content: "{{ https_star_private_key }}" 
+       crt_name: star.{{ base_domain }}.pem
```

Comment:
The template and vm environments do not define a restricted ip. This means that those parts of the haproxy config are not run in tests and not on simpler installations, so get less testing exposure. Would there be a way to also define a haproxy restricted ip and run the restricted apps on that?

Reply:
Yes, that's a good idea to test as well. I've added a second ip: https://github.com/OpenConext/OpenConext-deploy/pull/229/commits/f6b5c5b6ffbde0f178edbcdd6e53831349d3a8c7